controller/RestEligibility.php

```diff
@@ -367,14 +367,14 @@ private function getEligibilityFromRequest()
 
             return $this->propagate(new Eligibility($resource->getUri()));
         } catch (common_exception_NotFound $e) {
-            throw new common_exception_RestApi(__('Eligibility `%s` does not exist.', $eligibilityUri), 404);
+            throw new common_exception_NotFound(__('Eligibility `%s` does not exist.', $eligibilityUri));
         }
     }
 
     /**
      * Get delivery resource from request parameters
      * @return \core_kernel_classes_Resource
-     * @throws \common_exception_MissingParameter
+     * @throws common_exception_NotFound
```

Comment:
Missing leading `\`.
Or use `use` statements (see comments above).

Reply:
Leading `\` is not needed here since exception class is imported with `use` statement. The same for all other similar comments.